src/Deprecated/Engine/Shared/FrameworkLocationHelper.cs

```diff
@@ -66,8 +66,8 @@ internal static class FrameworkLocationHelper
         internal const string secondaryDotNetFrameworkSdkInstallKeyValueV40 = "CurrentInstallFolder";
         private const string dotNetFrameworkRegistryKeyV40 = dotNetFrameworkSetupRegistryPath + "\\v4\\Full";
         private static readonly GetDirectories getDirectories = new GetDirectories(Directory.GetDirectories);
-        
-        
+
+
```

Comment:
never two consecutive empty lines in .cs files, just remove one of them

Reply:
agreed

```suggestion
```